ui/src/ScholarReader.tsx

```diff
@@ -313,6 +321,28 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
         if (symbols.length >= 1) {
           const mathMl = await api.mathMlForArxivId(this.props.paperId.id);
           this.setMathMl(mathMl);
+
+          /**
+           * Build a mapping from symbol to all possible parent matches
+           */
+          const symbolMatchSet: SymbolMatchSet = {};
+          symbols.map(s => {
```

Comment:
should this just be a forEach instead of a map, given that the return value isn't used?

Reply:
yes good catch!